Microsoft.Toolkit.Uwp.SampleApp/App.xaml.cs

```diff
@@ -40,6 +40,29 @@ public App()
             Suspending += OnSuspending;
         }
 
+        protected override async void OnActivated(IActivatedEventArgs args)
+        {
+            await RunAppInitialization(null);
+
+            if (args.Kind == ActivationKind.Protocol)
+            {
+                try
+                {
+                    // Launching via protocol link
+                    var parser = DeepLinkParser.Create(args);
+                    var targetSample = await Sample.FindAsync(parser.Root, parser["sample"]);
+                    if (targetSample != null)
+                    {
+                        await Shell.Current?.NavigateToSampleAsync(targetSample);
```

Comment:
If Shell.Current is null, this will still throw an exception, even with the ? in place.